src/prioritized_cache.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "rocksdb/cache.h"
+#include "rocksdb/env.h"
+
+namespace rocksdb {
+namespace titandb {
+
+class PrioritizedCache {
```

Comment:
Sorry I didn't explain clearly again. What I want is something like this. Do you mind update the code? Thanks.
```
class PrioritizedCache: public Cache {
   public:
      PrioritizedCache(std::shared_ptr<Cache> cache, Cache::Priority priority)
          : cache_(cache), priority_(priority) {}

      Status Insert(..., Cache::Priority /*priority*/) override {
        return Insert(..., priority_);
      }

   private:
      std::shared_ptr<Cache> cache_;
      Cache::Priority priority_;
};
```

Reply:
With it  we can use the same underlying LRUCache for both rocksdb block_cache and titan blob_cache, but forcing block_cache to always insert with high-pri, and titan always insert with low-pri. This is because the data in rocksdb is the index for titan values, so they are more valuable and we want them to in cache more.

Reply:
Oh, I got it, I will update the code soon.